octavia_proxy/api/drivers/elbv3/driver.py

```diff
@@ -184,11 +204,14 @@ def listener_create(self, session, listener):
         if 'timeout_member_connect' in lattrs:
             lattrs['keepalive_timeout'] = \
                 lattrs.pop('timeout_member_connect')
+        if lattrs['tags']:
```

Comment:
same here... this fails with KeyError if the lattrs has no 'tags' key, rather use ``` if 'tags' in lattrs ```